src/Evolution/Systems/GeneralizedHarmonic/Equations.cpp

```diff
@@ -496,16 +497,58 @@ void UpwindFlux<Dim>::operator()(
 /// \endcond
 }  // namespace GeneralizedHarmonic
 
-// Explicit Instantiations
+// Explicit instantiations of structs defined in `Equations.cpp` as well as of
+// `partial_derivatives` function for use in the computation of spatial
+// derivatives of `gradients_tags`, and of the initial gauge source function
+// (needed in `Initialize.hpp`).
 /// \cond
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.tpp"
+
+using derivative_frame = Frame::Inertial;
+
+template <size_t Dim>
+using derivative_tags_initial_gauge =
+    tmpl::list<GeneralizedHarmonic::Tags::InitialGaugeH<Dim, derivative_frame>>;
+
+template <size_t Dim>
+using variables_tags_initial_gauge =
+    tmpl::list<GeneralizedHarmonic::Tags::InitialGaugeH<Dim, derivative_frame>>;
+
+template <size_t Dim>
+using derivative_tags =
+    typename GeneralizedHarmonic::System<Dim>::gradients_tags;
+
+template <size_t Dim>
+using variables_tags =
+    typename GeneralizedHarmonic::System<Dim>::variables_tag::tags_list;
```

Comment:
Use `db::get_variables_tags_list`, which can also handle prefixed variables tags. Not strictly necessary here, but may help someone who copies code around avoid a compiler error :)